gradle/libs.versions.toml

```
@@ -0,0 +1,233 @@
+mockito-junit-jupiter = { module = "org.mockito:mockito-junit-jupiter", version.ref = "org-mockito" }
+json-unit-fluent = "net.javacrumbs.json-unit:json-unit-fluent:2.35.0"
+truth = "com.google.truth:truth:1.1.3"
+truth-java8 = "com.google.truth.extensions:truth-java8-extension:1.1.3"
+# JUnit 5
+junit-jupiter = { module = "org.junit.jupiter:junit-jupiter", version.ref = "junit5" }
+junit-jupiter-engine = { module = "org.junit.jupiter:junit-jupiter-engine", version.ref = "junit5" }
+junit-vintage-engine = { module = "org.junit.vintage:junit-vintage-engine", version.ref = "junit5" }
+orchestrator = "androidx.test:orchestrator:1.4.1"
+# Hilt Testing
+hilt-android-testing = { module = "com.google.dagger:hilt-android-testing", version.ref = "hilt" }
+hilt-android-compiler = { module = "com.google.dagger:hilt-android-compiler", version.ref = "hilt" }
+# AndroidX Test
+androidx-test-runner = "androidx.test:runner:1.4.0"
+androidx-test-rules = "androidx.test:rules:1.4.0"
+androidx-test-ext = "androidx.test.ext:junit:1.1.3"
+# Espresso
+androidx-test-espresso-core = { module = "androidx.test.espresso:espresso-core", version.ref = "androidx-test-espresso" }
+androidx-test-espresso-intents = { module = "androidx.test.espresso:espresso-intents", version.ref = "androidx-test-espresso" }
+androidx-test-espresso-web = { module = "androidx.test.espresso:espresso-web", version.ref = "androidx-test-espresso" }
+androidx-test-espresso-contrib = { module = "androidx.test.espresso:espresso-contrib", version.ref = "androidx-test-espresso" }
+falcon = "com.jraska:falcon:2.2.0"
+screengrab = "tools.fastlane:screengrab:2.1.1"
+
+[bundles]
+kotlin-coroutines = [
+    "kotlin-coroutines-android",
+    "kotlin-coroutines-core",
+    "kotlin-coroutines-rx2",
+]
+android-ktx = [
+    "android-ktx-activity",
+    "android-ktx-core",
+    "android-ktx-fragment",
+    "android-ktx-preference",
+]
+android-lifecycle = [
+    "android-lifecycle-livedata",
+    "android-lifecycle-runtime",
+    "android-lifecycle-viewmodel",
+]
+androidx = [
+    "androidx-annotation",
+    "androidx-appcompat",
+    "androidx-browser",
+    "androidx-cardview",
+    "androidx-concurrent",
+    "androidx-constraintlayout",
+    "androidx-multidex",
+    "androidx-recyclerview",
+    "androidx-startup-runtime",
+    "androidx-swiperefreshlayout",
+    "androidx-viewpager2",
+    "androidx-work-runtime",
+]
+rx = [
+    "rx-android",
+    "rx-java",
+    "rx-kotlin",
+    "rx-relay",
+]
+networking = [
+    "networking-okhttp3-logging-interceptor",
+    "networking-retrofit-adapter-rxjava2",
+    "networking-retrofit-converter-jackson",
+    "networking-retrofit-converter-scalars",
+    "networking-retrofit-core",
+]
+apache-commons = [
+    "apache-csv",
+    "apache-text",
+    "apache-validator",
+]
+jackson = [
+    "jackson-annotations",
+    "jackson-core",
+    "jackson-databind",
+    "jackson-kotlin",
+]
+crash-analytics = [
+    "matomo",
+    "sentry-android",
+]
+unit-testing = [
```

I would rename it `testing` as we're also using libs for instrument tests
```suggestion
testing = [
```

---

Good call, I'll rename it.